libbeat/tests/system/test_dashboard.py

```diff
@@ -96,3 +161,28 @@ def get_kibana_host(self):
 
     def get_kibana_port(self):
         return os.getenv('KIBANA_PORT', '5601')
+
+    def create_kibana_space(self):
+        url = "http://" + self.get_kibana_host() + ":" + self.get_kibana_port() + \
+            "/api/spaces/space"
+        data = {
+            "id": "foo-bar",
+            "name": "Foo bar space"
+        }
+
+        headers = {
+            "kbn-xsrf": "1"
```

Comment:
out of curiosity: Why is this needed?

Reply:
It's for protecting against CSRF attacks: https://www.owasp.org/index.php/Cross-Site_Request_Forgery_(CSRF)_Prevention_Cheat_Sheet#Protecting_REST_Services:_Use_of_Custom_Request_Headers